src/app/components/login/login-toggle.jsx

```diff
@@ -37,14 +60,16 @@ function ChplLoginToggle() {
 
   return (
     <>
+    <ThemeProvider theme={theme}>
```

Comment:
Are you sure this is required? We don't need a ThemeProvider for every component, it only needs to exist once in a component tree, and having multiple ThemeProviders in the application is already giving us error/warning messages in the console

Reply:
> There are a lot of changes here that don't have anything to do with the feature under development. That makes it really hard for anyone reviewing either the code or the feature to make sure they can test all of the places that are touched. Can the changes in the theme be stripped down to just the things that are necessary for this feature?

Ill cut the theme changes down.